File: MultiFlex/API - New/JavaRestApi/src/main/java/org/acme/DTO/RegalDto.java
package org.acme.DTO;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class RegalDto {
    private Integer id;

    private String name;

    private Integer max_anzahl_faecher;

    //private Set<FachDto> faecher;

    private List<Integer> fach_ids;

    public RegalDto(Integer id, String name, Integer max_anzahl_faecher, List<Integer> fachIds) {
        this.id = id;
        this.name = name;
        this.max_anzahl_faecher = max_anzahl_faecher;
        this.fach_ids = fachIds;
    }

    public RegalDto() {
    }


}

File: MultiFlex/Frontend/Multiflex.Frontend/MultiFlex.Frontend.WebApp/Models/RegalDto.cs
using System.Collections.Generic;

namespace Multiflex.Frontend.WebApp.Models
{
    public class RegalDto
    {
        public int Id { get; set; }
        public string name { get; set; }
        public int max_anzahl_faecher { get; set; }
        public List<int> fach_ids { get; set; }

        
    }
}
